core/src/main/java/org/nervos/ckb/methods/response/CkbBannedAddresses.java

```diff
@@ -0,0 +1,13 @@
+package org.nervos.ckb.methods.response;
+
+import java.util.List;
+import org.nervos.ckb.methods.Response;
+import org.nervos.ckb.methods.type.BannedAddress;
+
+/** Created by duanyytop on 2019-07-26. Copyright © 2019 Nervos Foundation. All rights reserved. */
+public class CkbBannedAddresses extends Response<List<BannedAddress>> {
```

Comment:
The `BannedAddress` seems is `set_ban` parameter, but the `get_banned_addresses` response is different from this.

see here: https://github.com/nervosnetwork/ckb/tree/rc/v0.17/rpc#get_banned_addresses

Reply:
Oh. It's my fault.